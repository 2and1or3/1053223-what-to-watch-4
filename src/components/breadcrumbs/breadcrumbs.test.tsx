import * as React from "react";
import * as renderer from "react-test-renderer";

import BreadCrumbs from './breadcrumbs';

it(`Render BreadCrumbs component`, () => {
  const tree = renderer
    .create(
        <BreadCrumbs title = {`some title`}/>
    ).toJSON();

  expect(tree).toMatchSnapshot();
});

// npm run test.jest -- components/breadcrumbs/breadcrumbs.test
