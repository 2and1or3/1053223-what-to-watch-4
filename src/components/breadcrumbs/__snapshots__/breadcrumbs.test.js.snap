// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render BreadCrumbs component 1`] = `
<nav
  className="breadcrumbs"
>
  <ul
    className="breadcrumbs__list"
  >
    <li
      className="breadcrumbs__item"
    >
      <a
        className="breadcrumbs__link"
        href="movie-page.html"
      >
        some title
      </a>
    </li>
    <li
      className="breadcrumbs__item"
    >
      <a
        className="breadcrumbs__link"
      >
        Add review
      </a>
    </li>
  </ul>
</nav>
`;
